src/services/Auth/index.ts

```diff
@@ -0,0 +1,125 @@
+import * as dotenv from "dotenv";
+import { Request, Response } from "express";
+import { generateCode } from "../../services/generateCode";
+import bcrypt from "bcrypt";
+import { AuthRepository } from "../../Repository/Auth";
+import User from "../../models/userModel";
+import { findUser } from "../userExists";
+import { sendConfirmEMail } from "../nodemailerEmail";
+import { verifyPassword } from "../verifyPassword";
+import jwt from "jsonwebtoken";
+import joi from 'joi'
+import {tokens} from '../../models/tokenModel'
+import {digitalCode} from '../../services/digitalCode'
+import {passwordResetEMail} from '../nodemailerEmail'
+
+dotenv.config();
+
+const TOKEN_SECRET = String(process.env.TOKEN_SECRET);
+const auth = new AuthRepository();
+
+export const createUserService = async (req: Request, res: Response) => {
+    const code = generateCode();
+
+    const newUser = new User({
```

Comment:
Refactor all model calling to come from repository, Not calling model class directly